joomla/joomla-cms
# Master_Save Plugin

Pull Request for Issue # .

### Summary of Changes
create new Table "item_associations" by install
Put the columns 'associations' and 'not associations' together to one Column.
new created articles are master.
if associations with a Master, another articles are slave 


### Testing Instructions
create new articles without associations
make the association and go inside the " Multilingual Associations plugin" Area to see the changes

### Expected result



### Actual result



### Documentation Changes Required
